lib/config.js

```diff
@@ -17,6 +19,20 @@ module.exports.getConfig = async function getConfig({
     return config
   } catch (error) {
     log({ app, context, error, message: 'Invalid config file' })
+
+    if (error.isJoi && runnerIsActions()) {
```

Comment:
This is what it looks like now:

<img width="1358" alt="image" src="https://user-images.githubusercontent.com/876666/71783841-da54cf80-2fec-11ea-9723-fb9d6a5717ca.png">

Not perfect, because the config is in YAML and the config with error annotations is in JSON.

Alternatively I could log a table like so:

```
| Property   | Error message                               |
| ---------- | ------------------------------------------- |
| prerelease | "prerelease" must be a boolean              |
| sort-by    | "sort-by" must be one of [merged_at, title] |
```

Reply:
I'm convinced the JSON representation is confusing and noisy, I'm going to change it.

Reply:
As a table:

<img width="1358" alt="image" src="https://user-images.githubusercontent.com/876666/71784250-a29c5680-2ff1-11ea-8a24-3b2b7ebc98d2.png">

Much better if I say so myself :)